products/pdt/pdtlisten/src/lib.rs

```diff
@@ -171,13 +176,12 @@ pub async fn listen_psql(postgres_url: &str, api_url: &str) -> Result<()> {
 
     let provider = Provider::<Http>::try_from(api_url)?;
 
-    let mut stream = provider
-        .watch_blocks()
-        .await?
-        .map(|hash: H256| get_block_by_hash(hash, &provider))
-        .buffered(MAX_TASKS);
+    let mut stream = StreamExt::map(provider.watch_blocks().await?, |hash: H256| {
```

Comment:
Didn't we decide that `provider.watch_blocks()` doesn't work?

Reply:
this one was for `listen_psql`. Was planning to leave this one as it was and then implement it later. But might be easier to just do it now I guess?